docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,83 @@
+*******************************************************
+Storage Quota at /home/
+*******************************************************
+Staying under your ``/home/<username>`` quota is vital in preventing issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
```

Comment:
I just left off the important box because I don't know about listing the storage quota here as well on the storage page. Again, I could be swayed on this. Thank you!!